pkg/apis/core/types.go

```diff
@@ -3761,6 +3761,27 @@ type PodResourceClaimStatus struct {
 	ResourceClaimName *string
 }
 
+// PodExtendedResourceClaimStatus is stored in the PodStatus for the extended
+// resource requests backed by DRA. It stores the generated name for
+// the corresponding special ResourceClaim created by kube-scheduler.
```

Comment:
```suggestion
// the corresponding special ResourceClaim created by the scheduler.
```

There are other schedulers.

Actually, your API comments in the internal and external API are not in sync. I'm only going to comment on one version, please update in all places.